### PR TITLE
BTPANELS - Add plugins commonly used by team panels

### PR DESCRIPTION
**Pull request by rjacoby**
Adding a few simple plugins to help with general dev for the Panels.

- Bash
- Docker
- Markdown (with JavaFX plugin for display)

### CODE REVIEW

File: config/codestyles/Default.xml [hunk 1]
```diff
@@ -0,0 +1 @@
+<code_scheme name="Default" version="173" />
```
Comment (pgr0ss) [inline config/codestyles/Default.xml]: What's the deal with a new Default codestyle? What does this change?

File: config/options/statistics.main_menu.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  <component name="MainMenuCollector">
+    <counts>
+      <entry path="Build -&gt; Build Project" count="2" />
+      <entry path="Edit -&gt; Copy" count="1" />
+      <entry path="File -&gt; Exit" count="1" />
+      <entry path="File -&gt; Export Settings..." count="2" />
+      <entry path="File -&gt; Settings..." count="4" />
+    </counts>
+  </component>
+</application>
```
Comment (pgr0ss) [inline config/options/statistics.main_menu.xml]: I wonder if we should gitignore the statistics files.

File: config/codestyles/Default.xml (hunk 1 above)
Comment (rjacoby) [reply config/codestyles/Default.xml]: I think this is an artifact of the newer version of IntelliJ? These files were last committed under a diff version.

File: config/options/statistics.main_menu.xml (hunk 2 above)
Comment (rjacoby) [reply config/options/statistics.main_menu.xml]: Yeah, that seems reasonable.

**Pull request closed without merging**